project/Dependencies.scala

```diff
@@ -130,7 +132,7 @@ object Dependencies {
   // TODO check if `l ++=` everywhere expensive?
   val l = libraryDependencies
 
-  val actor = l ++= Seq(config, java8Compat.value)
+  val actor = l ++= Seq(config, java8Compat.value, javaAffinity)
```

Comment:
We have to keep the dependencies of `akka-actor` to a minimum so we can't add this dispatcher to `akka-actor`, but we can probably find another place for it when we have evaluated this.